<launch>
    <arg name = "vec_port"/>
    <arg name = "ultra_port"/>
    <node name="sensor_fusion" pkg="imu_driver" type="sensor_fusion.py" args="$(arg vec_port) $(arg ultra_port)" output = "screen"/>
    <node name= "convert_to_quaternion" type="convert_to_quaternion.py" pkg= "imu_driver"/>
    <!-- <node name= "listener" type="listener.py" pkg= "imu_driver"/> -->
</launch>